package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "54_list_of_odd_superstitions",
+  "version": "1.0.0",
+  "description": "��#\u0000 \u00005\u00004\u0000_\u0000L\u0000i\u0000s\u0000t\u0000_\u0000o\u0000f\u0000_\u0000O\u0000d\u0000d\u0000_\u0000S\u0000u\u0000p\u0000e\u0000r\u0000s\u0000t\u0000i\u0000t\u0000i\u0000o\u0000n\u0000s\u0000\r\u0000 \u0000",
```

Comment:
The `description` field currently contains non-readable characters. Consider updating this to a brief, readable description of your project. [important]